dts/bindings/pinctrl/nuvoton,npcx-pslctrl-def.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2021 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nuvoton, NPCX Default Power Switch Logic (PSL) input pads configurations
+
+compatible: "nuvoton,npcx-pslctrl-def"
+
+include: [base.yaml]
+
+properties:
+    psl-in-pads:
+        type: phandles
+        required: true
+        description: |
+            list of PSL input pads that are in charge of detecting the wake-up
+            signals and the related circuit will turn on core power supply
+            (VCC1) from standby power state (ultra-low power mode) later.
```

Comment:
ultra-low-power mode

Reply:
Done. Also fixed the other 'ultra-low power' strings to 'ultra-low-power' in the other files.